src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -1232,6 +1233,148 @@ describe('enrolment type selector', () => {
   );
 });
 
+describe('auto acceptance for enrolments', () => {
+  it('renders auto acceptance message when auto accept is set on', async () => {
+    renderComponent({
+      mocks: [getEventMockedResponse({ autoAcceptance: false })],
+    });
+    // Wait for form to have been initialized
+    await screen.findByTestId('time-and-location-form');
```

Comment:
I wonder if we could detect the form's existence based on something human readable? For instance its title or a field we are going to interact with.

https://testing-library.com/docs/queries/bytestid/
> In the spirit of the guiding principles, it is recommended to use this only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible. That said, they are way better than querying based on DOM structure or styling css class names. Learn more about data-testids from the blog post "Making your UI tests resilient to change"

Reply:
I don't mind - I just used this because it was used earlier also. I don't see anything wrong with this either. This form part starts with "tabs", but sure we could use first field of the first form section.